src/test/groovy/de/eddyson/tapestry/react/integration/AlertDemoSpec.groovy

```diff
@@ -24,7 +24,9 @@ class AlertDemoSpec extends JettyGebSpec {
     then:
     // TODO: this will need a custom ApplicationStatePersistenceStrategy
     // applicationStateManager.get(AlertStorage).alerts.size() == 1
-    helloWorld.displayed
+    waitFor {
```

Comment:
Would you mind adding new integration tests rather than modifying the existing ones?